src/coreclr/jit/lowerloongarch64.cpp

```diff
@@ -328,7 +328,8 @@ void Lowering::LowerBlockStore(GenTreeBlk* blkNode)
         }
         else
         {
-            blkNode->gtBlkOpKind = GenTreeBlk::BlkOpKindHelper;
+            blkNode->gtBlkOpKind =
+                blkNode->IsZeroingGcPointersOnHeap() ? GenTreeBlk::BlkOpKindLoop : GenTreeBlk::BlkOpKindHelper;
```

Comment:
It looks like the codegenloongarch64.cpp code is unfinished (with "TODO"). Should this change be commented out (left as it was before), and open an issue to fix the codegen and enable it?